### ISSUE TITLE
Add a command to allow users to send a message to committee members

### ISSUE THREAD

**Issue by MattyTheHacker** [CSSUoB/TeX-Bot-Py-V2#16]
The command would log the message sent to a channel, probably the TeX log channel.

Similar to the /helpop command of the ol' Minecraft days

**Comment by LMBishop** [comment]
/helpop hello i am from planet minecraft plz can i have op to review your minecraft server

**Comment by FrozenDude101** [comment]
How would this work differently from just @ Committee in the channels? (I presume it would @ Committe in the log channel?)

If this is making it anonymous, we've discussed anonymous feedback before and decided not to due to potential misuse.
So it should still say which member is asking for help in the log channel?

Most of the times anything that requires intervention happens (looking at you # serious-talk) there's already committee involved the in the discussion, and/or a couple messages in # discord to get other committee to keep an eye on it.

**Comment by MattyTheHacker** [comment]
It would be different because it allows a user to send a message to committee directly and keeping it private. It wouldn't be anonymous but it's mainly in the interest of giving people a way of sending a message to all of committee (instead of privately DMing one person)

**Comment by codemicro** [comment]
What about a ticket based system? ie. Someone can run /cmthelp or whatever and the bot makes a channel specifically for that person and cmt.

The bot could also "close" the channel when it's finished with by taking an archive, spitting it out in another channel and deleting the original one.

**Comment by MattyTheHacker** [comment]
> What about a ticket based system? ie. Someone can run /cmthelp or whatever and the bot makes a channel specifically for that person and cmt.
> 
> The bot could also "close" the channel when it's finished with by taking an archive, spitting it out in another channel and deleting the original one.
> 
> That way you can have multiple members of committee reply and talk to someone about a given thing.

Yeah sounds great! Sounds like an awful lot more work bi yeah sounds good.
